users/schema.py

```diff
@@ -367,6 +377,31 @@ def resolve_all_allergies(self, info, *args, **kwargs):
         return Allergy.objects.all().order_by("name")
 
 
+class KnightHoodQuery(graphene.ObjectType):
+    all_knighthoods = graphene.List(KnightHoodNode)
+
+    def resolve_all_knighthoods(self, info, *args, **kwargs):
+        return KnightHood.objects.prefetch_related("user").all().order_by("knighted_date")
+
+
+class KnightUserMutation(graphene.Mutation):
+    class Arguments:
+        user_id = graphene.ID()
+        knighted_date = graphene.Date(required=False)
+
+    user = graphene.Field(UserNode)
+
+    @staticmethod
+    @gql_has_permissions("users.change_user")
```

Comment:
spørs om man ikke vil ha en egen perm for dette. `"users.add_knighthood"` eller hva det nå enn blir

Reply:
aye, det er i boks og testet